apps/web/src/pages/SupportPage/components/SupportForm/SupportForm.tsx

```diff
@@ -52,35 +52,30 @@ const SupportFormSuccess = ({ onFormToggle }: SupportFormSuccessProps) => {
   const intl = useIntl();
   return (
     <section>
-      <Heading
-        level="h2"
-        size="h3"
-        data-h2-font-weight="base(400)"
-        data-h2-margin="base(0, 0, x1, 0)"
-      >
+      <Heading level="h2" size="h3" className="mt-0 mb-6 font-normal">
         {intl.formatMessage({
           defaultMessage: "We've received your message.",
           id: "iiEGjW",
           description: "Support form success title",
         })}
       </Heading>
-      <p data-h2-margin="base(x1, 0, x.5, 0)">
+      <p className="my-6">
         {intl.formatMessage({
           defaultMessage:
             "We'll do our best to get back to you with a response within the next two business days.",
           id: "OdjW9z",
           description: "Support form success paragraph one",
         })}
       </p>
-      <p data-h2-margin="base(x.5, 0, x1, 0)">
+      <p className="my-6">
         {intl.formatMessage({
           defaultMessage:
             "Please check your email for a summary of your submission.",
           id: "5r5XSh",
           description: "Support form success paragraph two",
         })}
       </p>
-      {/* <p data-h2-margin="base(x1, 0)">
+      {/* <p className="my-6">
```

Comment:
Out of scope, but why do we have this message? I couldn't find any open issues to add an FAQ.